docs/README.md

```diff
@@ -65,8 +71,8 @@ Commands:
   nrfu   Run NRFU against inventory devices
 ```
 
-!!! info
-    `username`, `password`, `enable`, and `enable-password` values are the same for all devices
+> [!NOTE]
```

Comment:
Seems GH markdown flavor is not supported by material for mkdocs. Rendering is not optimal

```
[!NOTE] username, password, enable-password, enable, timeout and insecure values are the same for all devices
```

Reply:
Yeah, the opposite is also true.. I chose the GitHub-compatible `[!NOTE]` annotation for the GitHub README page because `!!! info` is not rendered.
Maybe should we just use:
```
> username, password, enable-password, enable, timeout and insecure values are the same for all devices
```
So GitHub and mkdocs are happy ?

Reply:
Yes, let's do that ☮️